drivers/overlay/ov_network.go

```diff
@@ -613,7 +613,8 @@ func (n *network) initSandbox(restore bool) error {
 	sbox.InvokeFunc(func() {
 		nlSock, err = nl.Subscribe(syscall.NETLINK_ROUTE, syscall.RTNLGRP_NEIGH)
 		if err != nil {
-			err = fmt.Errorf("failed to subscribe to neighbor group netlink messages")
+			logrus.Errorf("Failed to subscribe to neighbor group netlink messages for overlay network %s in sbox %s: %v",
+				n.id, sbox.Key(), err)
```

Comment:
Agree. I kept the current logic, but I do not see a reason to let the network creation go through if it won't work properly.